apache/superset
# fix(dashboard): padding between rows within tabs

### SUMMARY
There should be padding between rows in a dashboard. Before, this did not work when the rows where within tabs. This is fixed here.
See this issue: #20303 

### BEFORE SCREENSHOT
<img width="1052" alt="Screenshot 2022-06-13 at 22 57 58" src="https://user-images.githubusercontent.com/16560193/173444283-a448a1d8-4217-44ef-998a-64bab1ee42b5.png">


### AFTER SCREENSHOT
<img width="1052" alt="Screenshot 2022-06-13 at 22 56 44" src="https://user-images.githubusercontent.com/16560193/173444094-e4447879-64d1-494f-bcf3-6ee67ca5ce9a.png">


### TESTING INSTRUCTIONS
* Insert two charts into a new dashboard
* Add a filter. When hovering over a filter, the selection box shows some space between the plots
* When you move the charts into tabs, there is no space between plots any more (see first screenshot)
* AFTER APPLYING CHANGES FROM PR:
* Now there is also some space when the charts are within tabs (see second screenshot).
